Feature/startrail 1145 deployment startrail-polygon2

wshino opened the pull request:
https://startbahn.slack.com/archives/CCSU382KT/p1636965097013400?thread_ts=1636931400.006000&cid=CCSU382KT

mitsu1124 commented:
@wshino cc @chatch 
Do you think this PR should be reviewed by someone?

chatch commented:
@wshino has this been deployed already? i think we already went to mainnet a couple of weeks ago with this right? if it's in mainnet and develop i think it's fine. qa doesn't really need this approach but prob. was useful to test the approach there right?

anyways, should we merge this or if we did it some other way that was already merged, just cancel it.

wshino commented:
@mitsu1124 @chatch I created this branch from qa, and since the qa branch had package-lock.json, I added it to the .gitignore and changed the base branch to develop.

anyway, now we are ready to merge into develop. sorry for the delay in replying.
